src/vl-cookie-consent-opt-in.js

```diff
@@ -0,0 +1,88 @@
+import { VlElement, define } from '/node_modules/vl-ui-core/dist/vl-core.js';
+import '/node_modules/vl-ui-checkbox/dist/vl-checkbox.js';
+import '/node_modules/vl-ui-form-message/dist/vl-form-message.js';
+import '/node_modules/vl-ui-form-grid/dist/vl-form-grid.js';
+
+/**
+ * VlCookieConsentOptIn
+ * @class
+ * @classdesc De cookie consent opt-in geeft de gebruiker om één specifiek soort van cookies te accepteren of te weigeren.
+ * 
+ * @extends VlElement
+ * 
+ * @property {boolean} data-vl-label - Attribuut bepaalt het label van de opt-in.
+ * @property {boolean} data-vl-description - Attribuut bepaalt de beschrijving van de opt-in.
+ * @property {boolean} data-vl-checked - Attribuut bepaalt of de opt-in standaard aangevinkt staat.
+ * @property {boolean} data-vl-mandatory - Attribuut bepaalt of de opt-in verplicht is en bijgevolg aangevinkt staat en niet wijzigbaar is.
+ * 
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-cookie-consent/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-cookie-consent/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-cookie-consent-opt-in.html|Demo}
+ * 
+ */
+export class VlCookieConsentOptIn extends VlElement(HTMLElement) {
+    static get _observedAttributes() {
+        return ['label', 'description', 'checked', 'mandatory'];
+    }
+
+    constructor() {
+        super(`
+            <style>
+                @import '/node_modules/vl-ui-form-grid/dist/style.css';
+                @import '/node_modules/vl-ui-form-message/dist/style.css';
```

Comment:
grid of form-message zitten toch niet in de shadow root hiervan?

Reply:
ah form-annotation wel zie ik hier verder

Reply:
of is "block" iets van vl-ui-form-grid dan?

Reply:
De form-grid dependency kan hier inderdaad verwijderd worden.